app/build.gradle

```diff
@@ -61,22 +61,48 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
 }
 
-
-task releaseClean {
+task ensureCleanRepo {
     doLast {
         if (!grgit.repository.jgit.status().call().clean) {
-            throw new GradleException('Git status is not clean, please commit or stash your changes!')
+            throw new GradleException('Git status is not clean, please stash your changes!')
         }
     }
 }
+task releaseClean(dependsOn: ensureCleanRepo) {
+    doLast {
+        def Clean = true
+        String headCommitMessage = grgit.head().shortMessage
+        while (headCommitMessage.contains("[gradle-release-task]")) {
+            Clean = false
+            println "Found git commit: $headCommitMessage"
+            if (headCommitMessage.indexOf("vitabu-") > -1) {
```

Comment:
To make the solution scalable to other Android apps (with different names, can you change "vitabu-" into a variable? For example by using the `applicationId` variable instead?